rosdep/python.yaml

```diff
@@ -3452,6 +3452,18 @@ python-scipy:
     yakkety_python3: [python3-scipy]
     zesty: [python-scipy]
     zesty_python3: [python3-scipy]
+python-scp:
+  debian: [python-scp]
+  fedora:
+    pip:
+      packages: [scp]
+  ubuntu:
+    xenial: [python-scp]
+    xenial_python3: [python3-scp]
```

Comment:
If you want a python3 version please add another key `python3-scp` that will resolve to it. The `_python3` distro suffixes were an experiment that we aren't using on newer platforms.